src/install/hashtopussy.sql

```diff
@@ -560,7 +563,12 @@ CREATE TABLE `User` (
   `lastLoginDate`      INT(11)          NOT NULL,
   `registeredSince`    INT(11)          NOT NULL,
   `sessionLifetime`    INT(11)          NOT NULL DEFAULT '600',
-  `rightGroupId`       INT(11)          NOT NULL DEFAULT '1'
+  `rightGroupId`       INT(11)          NOT NULL DEFAULT '1',
+  `yubikey`            INT(1)           NOT NULL DEFAULT '0',
+  `otp1`               VARCHAR(50)      NOT NULL,
```

Comment:
Is it correct that otp1 is varchar and the other three are ints?
As what I see from the other code, they should all have the same type.